src/content/docs/agile-handbook/appendices/ticket-best-practices.mdx

```diff
@@ -90,6 +90,17 @@ When you edit the site, include the Jira issue key (DOC-1234, for example) in yo
       </td>
     </tr>
 
+        <tr>
+      <td>
+        User story
+      </td>
+
+      <td>
+        *According to [Atlassian](https://www.atlassian.com/agile/project-management/user-stories) "A user story is an informal, general explanation of a software feature written from the perspective of the end user. Its purpose is to articulate how a software feature will provide value to the customer. 
+        *User stories are often expressed in a simple sentence, structured as follows: 'As a [persona], I [want to], [so that].'"'
```

Comment:
I don't understand why this line ends with '"' Is that intentional?
Also, if you want lines 99 and 100 to be bullets, there should be a space after the * beginning those lines. Hope this helps!

Reply:
nope! not intentional! thank you!